packages/lit-element/src/index.ts

```diff
@@ -0,0 +1,10 @@
+export * from '@lit/reactive-element';
+export * from 'lit-html';
+export * from './lit-element.js';
+
+// TODO: link to docs on the new site
+console.warn(
+  "The main 'lit-element' package is deprecated. " +
+    "Please import from the 'lit' package: 'lit' and 'lit/decorators.ts' " +
+    "or import from 'lit-element/lit-element.ts'."
```

Comment:
```suggestion
  "The main 'lit-element' module entrypoint is deprecated. " +
    "Please update your imports to the 'lit' package: 'lit' and 'lit/decorators.js' ".
```
Don't want it to read like `LitElement` is deprecated.